tests/unifiedPlatform/pds2/pds_dataservice_test.go

```diff
@@ -87,6 +88,7 @@ var _ = Describe("{UpgradeDataServiceImage}", func() {
 
 			stepLog := "Running Workloads after upgrading the ds image"
 			Step(stepLog, func() {
+				WorkflowDataService.WorkloadGenParams.TableName = "wltesting" + RandomString(3)
```

Comment:
Could we add this as part of WorkflowDataService.RunDataServiceWorkloads, so that we dont have to modify all tests, otherwise I think in resiliency tests also these changes might be requierd.

Reply:
This line is not required anymore, I have already added it in workflows 👍